packages/forms/src/Components/BaseFileUpload.php

```diff
@@ -99,6 +101,13 @@ protected function setUp(): void
             // An ugly mess as we need to support both Flysystem v1 and v3.
             $storageAdapter = method_exists($storage, 'getAdapter') ? $storage->getAdapter() : (method_exists($storageDriver = $storage->getDriver(), 'getAdapter') ? $storageDriver->getAdapter() : null);
             $supportsTemporaryUrls = method_exists($storageAdapter, 'temporaryUrl') || method_exists($storageAdapter, 'getTemporaryUrl');
+            
+            try {
```

Comment:
Instead of catching the exception & logging, let's just use an `exists()` check for the file to make sure it exits before we make the `getVisibility()` check.